theano/gpuarray/tests/test_pickle.py

```diff
@@ -19,7 +19,7 @@
 from theano.misc.pkl_utils import CompatUnpickler
 
 try:
-    from . import config  # noqa
+    from .config import mode_with_gpu  # noqa
```

Comment:
done

On Thu, May 12, 2016 at 12:33 PM, abergeron <notifications@github.com>
wrote:

> In theano/gpuarray/tests/test_pickle.py
> <https://github.com/Theano/Theano/pull/4503#discussion_r63052587>:
>
> > @@ -19,7 +19,7 @@
> >  from theano.misc.pkl_utils import CompatUnpickler
> >
> >  try:
> > -    from . import config  # noqa
> > +    from .config import mode_with_gpu  # noqa
>
> Then we could use from . import config as _
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/Theano/Theano/pull/4503/files/c075d232b0ea3a70d1412808e8ffa11979e6fede#r63052587>
>